systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlConfigurationST.java

```diff
@@ -256,6 +261,23 @@ void testConfigurationPerformanceOptions(ExtensionContext extensionContext) thro
                 assertThat(containerConfiguration, hasEntry(key, value.toString())));
     }
 
+    @ParallelNamespaceTest
+    void testKafkaRebalanceAutoApprovalMechanism(ExtensionContext extensionContext) {
```

Comment:
I don't follow why you moved this here. This test seems to not be related to the other tests in this class. It tests the Cruise Control API (which is the KafkaRebalance resource). So it should have stayed where it was?

(Leaving aside that at least some tests in this class look like they should be unit tests and not system tests)

Reply:
> I don't follow why you moved this here. This test seems to not be related to the other tests in this class. It tests the Cruise Control API (which is the KafkaRebalance resource). So it should have stayed where it was?

Whoops, I meant to move that in the `CruiseControlST` class. It seemed that the `CruiseControlST` class has tests related to the `KafkaRebalance` resource where as the `CruiseControlApiST` class seemed to contain tests for hitting/testing the CC API directly.

> (Leaving aside that at least some tests in this class look like they should be unit tests and not system tests)

I'll investigate this class to to see what can be removed

Reply:
Yeah, that makes sense I guess.